knative/knative-serving-crds/overlays/application/application.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: knative-serving-crds
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: knative-serving-crds
+      app.kubernetes.io/instance: knative-serving-crds-v0.8.0
```

Comment:
Why v0.8.0?

Reply:
Looks like that's the version of knative we are using.
https://github.com/knative/serving/releases